File: GTTS/INL.VettingService.Models/PersonVettingStatus_Item.cs
using System;

namespace INL.VettingService.Models
{
    public class PersonVettingStatus_Item : IPersonVettingStatus_Item
    {
        public long PersonID { get; set; }
        public long? TrainingEventID { get; set; }
        public int VettingBatchStatusID { get; set; }
        public string BatchStatus { get; set; }
        public int VettingPersonStatusID { get; set; }
        public string PersonsVettingStatus { get; set; }
        public DateTime? DateLeahyFileGenerated { get; set; }
        public DateTime? VettingBatchStatusDate { get; set; }
        public DateTime? ExpirationDate { get; set; }
        public bool RemovedFromVetting { get; set; }
        public DateTime? VettingStatusDate { get; set; }
        public int VettingBatchTypeID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/person-vetting-status_item.ts


import { IPersonVettingStatus_Item } from './iperson-vetting-status_item';

export class PersonVettingStatus_Item implements IPersonVettingStatus_Item {
  
	public PersonID: number = 0;
	public TrainingEventID?: number;
	public VettingBatchStatusID: number = 0;
	public BatchStatus: string = "";
	public VettingPersonStatusID: number = 0;
	public PersonsVettingStatus: string = "";
	public DateLeahyFileGenerated?: Date;
	public VettingBatchStatusDate?: Date;
	public ExpirationDate?: Date;
	public RemovedFromVetting: boolean = false;
	public VettingStatusDate?: Date;
	public VettingBatchTypeID: number = 0;
  
}


